schema/files/TransactionsFile.schema.json

```diff
@@ -55,7 +55,7 @@
             "$ref": "https://raw.githubusercontent.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/main/schema/objects/transactions/issuance/ConvertibleIssuance.schema.json"
           },
           {
-            "$ref": "https://raw.githubusercontent.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/main/schema/objects/transactions/issuance/PlanSecurityIssuance.schema.json"
```

Comment:
@pjohnmeyer, what do you think, should we keep both $ids in the TransactionsFile items permitted $refs? Seems like yes, even if, for validation purposes, I don't think it should be strictly necessary. Someone may be referencing or parsing this list and be surprised by the omission of the PlanSecurity... $ids.

Reply:
If it validates either way I'm not sure I care.